src/MonologCreator/Processor/ExtraFieldProcessor.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+
+namespace MonologCreator\Processor;
+
+/**
+ * Class ExtraFieldProcessor
+ *
+ * Allows adding additional high-level or special fields to the log output.
+ *
+ * @package MonologCreator\Processor
+ * @author Sebastian Götze <s.goetze@bigpoint.net>
+ */
+class ExtraFieldProcessor implements \Monolog\Processor\ProcessorInterface
+{
+    /**
+     * Array to hold additional fields
+     *
+     * @var array
+     */
+    protected $_extraFields = array();
```

Comment:
Please don't use `_` in variable names as prefixes to mark them "internal". This is discouraged in the PHP community, as by now IDE's will honor the visibility definitions in front of the variable and makes changing the visibility hard to read in code, as you have variable renames all over the place.

Why is this variable protected? Are you extending from this class and need it in the lower class?

Reply:
Will remove the _.

There are no current plans for extending the class. But if it was to be extended, $extraFields would be crucial to that class, since it is what this Processor is about. ;)

Reply:
We are normally developing for the "needs" we have right now. So for now I would say make it private.